concepticondata/concepticon.tsv

```diff
@@ -3174,3 +3174,55 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3181	MUSTARD	Food and drink	It can be a yellow sauce with a strong taste or a plant which is used to make mustard sauce.	Person/Thing	
+3182	MARRIAGE OR WEDDING	Kinship	The ceremony of marrying someone or the state of being married.	Person/Thing	
+3183	BREATH OR BREATHE	The body	The air that goes into one's lungs or the act of breathing	Other	
+3184	BE ALIVE OR LIFE	The body	The act of being alive or the life itself.	Other	
+3185	BE DEAD OR DIE OR KILL	The body	To be not alive or to cease to live or to cause the end of a life.	Other	
+3186	BE DEAD OR DEATH OR DIE	The body	To be not alive or the end of a life or to cease to live.	Other	
+3187	DROWNED	The body	To have perished by suffocation in water.	Property	
+3188	MIGHTY OR POWERFUL OR STRONG	Emotions and values	Holding much (political, social, ...) power or being done using a huge amount of energy or having great physical strength.	Property	
+3189	STAB TO DEATH	The body	To kill by stabbing with a knife.	Action/Process	
+3190	BROKEN (NOT WORKING)	Basic actions and technology	Not capable of working.	Property	
+3191	BROKEN (SHATTERED)	Basic actions and technology	Broken into pieces.	Property	
+3192	BRONZE OR COPPER	The physical world	Chemical element with symbol Cu and atomic number 29 or an alloy made from it.	Person/Thing	
+3193	PAINT (WALL)	Basic actions and technology	To apply color to a wall.	Action/Process	
+3194	PAINT (WALL) OR PAINTING	Basic actions and technology	To apply color to a wall or to make a painting or drawing.	Action/Process	
+3195	DAMAGE OR INJURE	Possession	To put a thing in bad condition by making it suffer some damage or to cause physical harm to a living creature.	Action/Process	
+3196	BE LEFT	Spatial relations	To continue to be in the same place after others have left or been removed.	Property	
+3197	SUSPICION	Cognition	An assumption without direct evidence.	Person/Thing	
+3198	NOTHING OR ZERO	Miscellaneous function words	Not anything or the cardinal number that denotes no quantity or amount at all.	Other	
+3199	GUESS (SUCCESSFULLY)	Cognition	To succeed to know, or discover, by way of surmises.	Action/Process	
+3200	COUNTRYSIDE	The physical world	The region outside of a city or town.	Person/Thing	
+3201	HINDER OR PREVENT	Social and political relations	To make it difficult for something to occur or to fully stop it from occurring.	Action/Process	
+3202	CANNON	Warfare and hunting	A bullet firing weapon mounted on a carriage.	Person/Thing	
+3203	CANNON OR GUN	Warfare and hunting	A bullet-firing weapon mounted on a carriage or to be held in the hands or arms.	Person/Thing	
+3204	INSULT	Social and political relations	To use swear-words towards someone.	Action/Process	
+3205	GRAVEN IMAGE	Religion and belief	A representation of a deity who is considered to be false.	Person/Thing	
+3206	ELF OR FAIRY	Religion and belief	A human-like, often youthful-looking being with supernatural powers.	Person/Thing	
```

Comment:
I see we have the "often", but this sounds a bit Tolkienesque -- even medieval North European elves and fairies where not necessarily youthful, human-like, or had supernatural powers. I'd use something like mythical beings similar but distinct from humans, related to nature, etc, and usually with supernatural or magic powers.

Reply:
Nothing linked? Sorry, but where does it come from?

Reply:
Don't know. Some lists are linked to ELF OR FAIRY and write "elf, fairy" or similar, and I guess they were originally linked to either ELF or FAIRY, or those concepts were added prematurely in case those single meanings would come up one day.